src/components/dialogs/two-windings-transformer/phase-tap-changer-pane.js

```diff
@@ -234,6 +217,12 @@ const PhaseTapChangerPane = (props) => {
 
     const handleEditCell = useCallback(
         (rowData, newVal) => {
+            setLineEdit({
+                oldValues: {},
+                newValues: {},
+                id: rowData.rowIndex,
+                errors: new Map(),
+            });
             const parsedVal = parseInt(newVal);
```

Comment:
Maybe change parseInt to parseFloat ? Otherwise it's not possible to edit ratio or shift angle fields with decimal values

Reply:
This said I think we ought to keep a truncated value for the columns expressed in percentage

Reply:
done.